ansible/checker.yml

```diff
@@ -0,0 +1,48 @@
+# This playbook deploys a simple stand alone checker for SAP/HANS.
```

Comment:
Let's remove or update this comment. It looks outdated

Reply:
I would rename the file name from `checker.yml` to `check.yml`. This runs an action, so an imperative verb looks more appropiate